run `make build` under `erc20-market-cap` emits errors

note i have run `make erc20-price-spkg` under root, but still:

```
➜  erc20-market-cap git:(bump-rust-toolchain) make build
cargo build --target wasm32-unknown-unknown --release
   Compiling substreams-helper v0.0.2 (/Users/yiksanchan/source/0xbe1/web3/messari-substreams/substreams-helper)
error[E0463]: can't find crate for `std`
  |
  = note: the `wasm32-unknown-unknown` target may not be installed
  = help: consider downloading the target with `rustup target add wasm32-unknown-unknown`

error[E0463]: can't find crate for `std`
 --> substreams-helper/src/erc20.rs:1:5
  |
1 | use std::ops::Div;
  |     ^^^ can't find crate
  |
  = note: the `wasm32-unknown-unknown` target may not be installed
  = help: consider downloading the target with `rustup target add wasm32-unknown-unknown`

error[E0463]: can't find crate for `std` which `ethabi` depends on
 --> substreams-helper/src/erc20.rs:3:5
  |
3 | use ethabi::ethereum_types::U256;
  |     ^^^^^^ can't find crate
  |
  = note: the `wasm32-unknown-unknown` target may not be installed
  = help: consider downloading the target with `rustup target add wasm32-unknown-unknown`

error[E0463]: can't find crate for `std` which `substreams` depends on
 --> substreams-helper/src/erc20.rs:4:5
  |
4 | use substreams::Hex;
  |     ^^^^^^^^^^ can't find crate
  |
  = note: the `wasm32-unknown-unknown` target may not be installed
  = help: consider downloading the target with `rustup target add wasm32-unknown-unknown`
...
```

--- a/erc20-market-cap/rust-toolchain.toml
+++ b/erc20-market-cap/rust-toolchain.toml
@@ -1,0 +1,4 @@
+[toolchain]
+channel = "1.64.0"
+components = [ "rustfmt" ]
+targets = [ "wasm32-unknown-unknown" ]

